extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/runtime/src/main/java/io/quarkus/resteasy/reactive/jackson/CustomSerialization.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.resteasy.reactive.jackson;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.lang.reflect.Type;
+import java.util.function.BiFunction;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+
+/**
+ * Annotation that can be used on RESTEasy Reactive Resource method to allow users to configure Jackson serialization
+ * for that method only, without affecting the global Jackson configuration.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.METHOD })
+public @interface CustomSerialization {
+
+    /**
+     * A {@code BiFunction} that converts the global {@code ObjectMapper} and type for which a custom {@code ObjectWriter} is
+     * needed
+     * (this type will be a generic type if the method returns such a generic type) and returns the instance of the custom
+     * {@code ObjectWriter}.
+     *
+     * Quarkus will construct one instance of this {@code BiFunction} for each JAX-RS resource method that is annotated with
+     * {@code CustomSerialization} and once an instance is created it will cached for subsequent usage by that resource method.
```

Comment:
```suggestion
     * {@code CustomSerialization} and once an instance is created it will be cached for subsequent usage by that resource method.
```